src/benchmark/Akka.Benchmarks/Streams/SimpleStreamingComparisonBenchmarks.cs

```diff
@@ -0,0 +1,99 @@
+﻿#region copyright
+//-----------------------------------------------------------------------
+// <copyright file="SimpleStreamingComparisonBenchmarks.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2018 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2018 .NET Foundation <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+#endregion
+
+using Akka.Actor;
+using Akka.Benchmarks.Configurations;
+using Akka.Streams;
+using Akka.Streams.Dsl;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Engines;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Threading.Tasks.Dataflow;
+
+namespace Akka.Benchmarks.Streams
+{
+    [Config(typeof(MonitoringConfig))]
+    [SimpleJob(RunStrategy.Monitoring, launchCount: 3, warmupCount: 3, targetCount: 3)]
+    public class SimpleStreamingComparisonBenchmarks
```

Comment:
Could we add Orleans Streams in the future or aren’t they made for such „simple“ scenarios ?

Reply:
Initializing Orleans runtime itself is not a "simple" scenario - also I have an impression, that Orleans Streams name is a bit missleading, see: https://twitter.com/Horusiath/status/1046032430883328007
What I think, we could actually add here is [Nessos.Streams](http://nessos.github.io/Streams/) - they benchmarks claim, that they are faster than LINQ, so could be a nice for a baseline comparison (as where can be actually be).